README.md

```diff
@@ -1 +1,237 @@
+blokchain. The process above will automatically fetch some tokens from a faucet
+**under development**. For production environments, you should contact us (Vocdoni)
+for a byte64 string faucet, and specify it when creating your account:
+
+~~~ts
+(async () => {
+  const info = await client.createAccount({
+    faucetPayload: "<b64string>"
+  })
+})();
+~~~
+
+Accounts also require Vocdoni tokens in order to create new processes.
+
+You can check the balance thanks to the previous methods (`createAccount` and/or
+`fetchAccountInfo`) and, under development, you can request new tokens thanks
+to the included faucet:
+
+~~~ts
+(async () => {
+  const info = await client.createAccount()
+  if (info.balance === 0) {
+    await client.collectFaucetTokens()
+  }
+})();
+~~~
+
+> Note: the `collectFaucetTokens` method only works under development environment
+> and for accounts with not enough tokens to create new processes.
+
+### Creating a voting process
+
+After you successfully registered your account against Vocdoni's blockchain, you
+can start creating processes.
+
+A process will require a census of people who will vote it. Let's start creating
+it:
+
+~~~ts
+const census = new PlainCensus()
+census.add(address)
+// "address" is a valid ethereum-alike address
+~~~
+
+When creating the process, we recommend you adding yourself to the census:
+
+~~~ts
+const census = new PlainCensus()
+// signers return addresses as promises, hence the usage of async here
+(async () => {
+  census.add(await signer.getAddress())
+})();
+~~~
+
+After you got all the addresses for your census, you may as well create the
+process instance:
+
+~~~ts
+const election = new Election({
+  title: 'Election title',
+  description: 'Election description',
+  // a header image for your process (this is for example purposes; avoid using random sources)
+  header: 'https://source.unsplash.com/random/2048x600',
+  startDate: new Date('2023-01-01 00:00:00'),
+  endDate: new Date('2023-01-23 23:23:23'),
+  census,
+})
+~~~
+
+Of course you will need some questions in this voting process, how would people
+vote otherwise?
+
+~~~ts
+election.addQuestion('Ain\'t this process awesome?', [
+  {
+    title: 'Yes',
+    value: 0,
+  },
+  {
+    title: 'No',
+    value: 1
+  }
+])
+~~~
+
+> If you're a developer, maybe the value set to zero in Yes (and viceversa)
+> confuses you. Note that this is a mapping of values; people voting on Yes will
+> properly set the value as specified (zero in this case), thus printing the
+> results as you expect.
+
+You can finally confirm the transaction in the blockchain by just calling
+`createElection`:
+
+~~~ts
+(async () => {
+  const id = await client.createElection(election)
+  console.log(id) // will show the created election id
+})();
+~~~
+
+The election id you got there will be the one you use to access the election.
+After a few minutes of creating it, you should be able to check it on
```

Comment:
I would avoid `minutes`. Let's create the process immediately.

Reply:
The process is created immediately... but truth is it does not appear in the explorer after a few minutes 😅